ui/README.md

```diff
@@ -1,3 +1,13 @@
-## Mobile Control Panel UI
+# Mobile Control Panel UI
 
-This currently integrates into the OpenShift UI as an extension.
\ No newline at end of file
+
+## Local Installation
+
+Start openshift, configure it for MCP Extension development, and install the mobile apiserver
+
+```
+./install.sh
+```
+
+The MCP Extension should now be visible in the OpenShift Web Console after the 'origin' container has restarted.
+Visit https://127.0.0.1:8443 to see the Console.
```

Comment:
Would it be worth saying. Use oc to create some mobile apps and you should see them in the ui? To show that integration point?